tests/integration/security/authorization_test.go

```diff
@@ -1212,28 +1213,80 @@ func TestAuthorization_Audit(t *testing.T) {
 					ExpectAllowed: expectAllowed,
 				}
 			}
-			cases := []rbacUtil.TestCase{
-				newTestCase(b, "/allow", true),
-				newTestCase(b, "/audit", false),
-				newTestCase(c, "/audit", true),
-				newTestCase(c, "/deny", false),
-				newTestCase(d, "/audit", true),
-				newTestCase(d, "/other", true),
-			}
 
-			args := map[string]string{
-				"Namespace":     ns.Name(),
-				"RootNamespace": istio.GetOrFail(t, t).Settings().SystemNamespace,
-			}
+			for _, a := range []echo.Instances{apps.A.Match(echo.Namespace(ns.Name())), apps.VM.Match(echo.Namespace(ns.Name()))} {
```

Comment:
a is the source workload sending the request right? I don't think replacing it with vm has any benefits, the destination pod should be replaced with vm as the audit log happens only on the server side.

Reply:
I have removed the for loop, now only a is chosen as sender.